src/Symfony/Component/Console/Tests/Helper/ProgressBarTest.php

```diff
@@ -23,10 +23,13 @@
  */
 class ProgressBarTest extends TestCase
 {
+    private const FIFTY_MS = 50001;
```

Comment:
Variables whose names describe their contents are not future proof :)  If we change this in the future to `40001`, the code will be confusing. We can rename this to `REDRAW_INTERVAL`, `REDRAW_TIME`, `FRAME_DURATION` or anything else we like. Thanks.